src/base/TilesRendererBase.js

```diff
@@ -390,6 +392,9 @@ export class TilesRendererBase {
 
 		tile.__lastFrameVisited = - 1;
 
+		this.invokeAllPlugins( plugin => {
+			plugin !== this && plugin.preprocessNode && plugin.preprocessNode(tile, uri);
```

Comment:
We should pass the parent tile in to this function the same way it is in the main function.

Reply:
Yes, although i'm also passing the `uri` instead of the `tileSetDir`. Maybe this hook could be cleaner :/

Reply:
Oh I see, now. Hm. It should just take the same arguments at the parent function but I see that you need to keep the  original content uri definition since it's used to expand the content uris. Maybe it's best if we don't modify the content uri in the preprocessNode function - but lets deal with that in another PR and fix the arguments passed here.